solutions/swb-reference/jest.config.js

```diff
@@ -24,8 +24,18 @@ async function init() {
     fs.readFileSync(join(__dirname, `integration-tests/config/${stage}.yaml`), 'utf8')
   );
 
+  const apiStackOutputs = JSON.parse(
+    fs.readFileSync(join(__dirname, `src/config/${process.env.STAGE}.json`), 'utf8')
+  );
+  const apiStackName = Object.entries(apiStackOutputs).map(([key, value]) => key)[0]; //output has a format { stackname: {...props} }
+  const outputs = apiStackOutputs[apiStackName];
+
+  const mainAccountId = outputs.dynamoDBTableOutput.split(':')[4];
```

Comment:
Can we return the main Account Id as parameter as well ?, I am not sure but I think we actually have the account number [here](https://github.com/awslabs/solution-spark-on-aws/blob/main/solutions/swb-reference/src/SWBStack.ts#L500:~:text=resources%3A%20%5B%60arn%3Aaws%3Acatalog%3A%24%7Bthis.region%7D%3A%24%7Bthis.account%7D%3Aportfolio/*%60%5D)  and we can just set it as output

Reply:
I'm not sure if we need to export it for integration tests since we can derive it from one of many other stack outputs, like we're doing here.